test/test_fields.py

```diff
@@ -268,6 +314,7 @@ class Article(MongoModel):
         self.assertIs(article.tags, article.tags)
 
         article.tags.append('foo')
+        self.assertEqual(article.tags, ['foo'])
```

Comment:
Nice catch.

Reply:
Tracked in [PYMODM-76](https://jira.mongodb.org/browse/PYMODM-76)